DOCUMENTATION.md

```diff
@@ -0,0 +1,150 @@
+<h1 align="center">
+Renovate Configuration
+</h1>
+
+<br>
+
+#### While we already include a broad base configuration by default you can also add to, and extend your own repo's configuration to meet your project's needs. 
+
+<br>
+
+#### By default the base configuration covers most of the common package dependency standards found in popular languages. However if you are interested in customizing further for your language of choice please consult the [Renovate Docs](https://docs.renovatebot.com/). 
+
+<br>
+
+<h2 align="center">
+*** NOTE *** 
+<br>
+***** DO NOT OVERRIDE BASE CONFIGURATIONS LOCALLY *****
+</h2>
+
+  If your team feels the base configuration does not meet your needs please come and contact us in Slack in `#platform-core-services`! We are a lovely bunch and are more than happy to help fix your pain points! We can also explain the reasoning behind the settings chosen and help you find a compromise without sacrificing security. We also welcome PRs against the `cds-snc/renovate-config` repo if you find a bug or have a suggestion for a good base setting other teams might benefit from!
+
+# 
+
+## Table of Contents:
+1. [Base Configuration](#base-configuration)
+2. [Anatomy of a PR](#anatomy-of-a-pr)
```

Comment:
nit: noticed that a few of these named anchors don't work.

Reply:
Fixed. Good to note: you can't use fancy HTML _and_ hyperlinks in markdown.